spec/api/datastore/entities/v1/crime_application_spec.rb

```diff
@@ -20,12 +20,12 @@
   end
 
   let(:id) { SecureRandom.uuid }
-  let(:submitted_at) { 3.days.ago }
+  let(:submitted_at) { 3.days.ago.to_json }
```

Comment:
Maybe `#iso8601(3)` is cleaner? The json one produces escaped quotes in the string.

```ruby
irb(main)> x = 3.days.ago
irb(main)> x.to_json
=> "\"2023-05-22T13:24:55.611Z\""
irb(main)> x.iso8601(3)
=> "2023-05-22T13:24:55.611Z"
```

But now I'm unsure the test is working as expected? No matter what you set in the let, it always passes 😛

Reply:
Nice. Much cleaner will change.